src/main/java/hudson/plugins/throttleconcurrents/ThrottleJobProperty.java

```diff
@@ -24,20 +30,11 @@
 import java.util.Map;
 import java.util.WeakHashMap;
 import java.util.concurrent.CopyOnWriteArrayList;
-import javax.annotation.CheckForNull;
-import javax.annotation.Nonnull;
-import jenkins.model.Jenkins;
 
-import net.sf.json.JSONObject;
-
-import org.kohsuke.stapler.DataBoundConstructor;
-import org.kohsuke.stapler.QueryParameter;
-import org.kohsuke.stapler.StaplerRequest;
-
-public class ThrottleJobProperty extends JobProperty<AbstractProject<?,?>> {
+public class ThrottleJobProperty extends JobProperty<Job<?,?>> {
     // Moving category to categories, to support, well, multiple categories per job.
     @Deprecated transient String category;
-    
+
```

Comment:
Sure, I will get rid of them. Actually, I removed them manually before commit, but IDE put them again :)